### DISCUSSION TITLE
Moore's Law Graphs!

### DISCUSSION THREAD

**Discussion by gunnardesantisUCONN** [cooperrc/computational-mechanics#197] [Show and tell]
<img width="347" alt="image" src="https://github.com/cooperrc/computational-mechanics/assets/134865112/b2a3e5e7-fb82-4b71-8105-2815a06488da">

Hey Y'all - now that HW 2 is turned in, I was wondering if anyone else got something like this for the Moore's Law problem?

Also would anyone want to grind out some of the 3255 material this weekend at a library?

Gunnar

**Comment by cooperrc** [comment]
It looks like the "moore's law fit" is still on a linear scale or its in limbo between the two. 

You should be able to do the fit with 

`fit = np.polyfit(year, np.log(count), 1)`

then plot it with

`plt.plot(year, np.exp(np.polyval(fit, year)))`
